windows/ReactTestApp/MainPage.h

```diff
@@ -22,15 +24,20 @@ namespace winrt::ReactTestApp::implementation
     private:
         ::ReactTestApp::ReactInstance reactInstance_;
 
-        void SetReactComponentName(Windows::Foundation::IInspectable const &,
-                                   Windows::UI::Xaml::RoutedEventArgs);
+        void LoadReactComponent(std::string const &appKey,
+                                std::optional<std::map<std::string, std::any>> const &initialProps);
+        void
+        WriteInitialProperties(std::optional<std::map<std::string, std::any>> const &initialProps);
```

Comment:
Should this be `Set*` instead?
```suggestion
        void
        SetInitialProperties(std::optional<std::map<std::string, std::any>> const &initialProps);
```